README.md

```diff
@@ -87,7 +87,9 @@ OPTIONS:
 
 SUBCOMMANDS:
     build    Build your app in release mode
+    dev      Starts the dev mode
     help     Print this message or the help of the given subcommand(s)
+    init     Generates dockerfile and docker compose file with the init command
```

Comment:
```suggestion
    init     Generates Dockerfile and Docker Compose files
```

Reply:
I just copied the help result. We need to change the comment in the command enum to change it: https://github.com/lenra-io/lenra_cli/blob/cc4bc8dc886d8811b81a3d90fd52c84658a94636/src/cli/mod.rs#L37